src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml.cs

```diff
@@ -306,5 +306,15 @@ void ViewCommentsClick(object sender, RoutedEventArgs e)
             }
             catch { }
         }
+
+        void OpenHyperlink(object sender, ExecutedRoutedEventArgs e)
+        {
+            var url = e.Parameter.ToString();
+
+            if (!string.IsNullOrEmpty(url))
+            {
+                VisualStudioBrowser.OpenUrl(new Uri(url));
```

Comment:
As @shana said in https://github.com/github/VisualStudio/pull/1133#discussion_r133485930 - these links are created by us, so they shouldn't be malformed.